rfcs/20200612-stream-executor-c-api.md

```diff
@@ -110,21 +114,15 @@ typedef TF_Status* (*TF_StatusCallbackFn)(void*);
 
 typedef struct SE_PlatformId {
  size_t struct_size;
+ void* ext;
  void* id;  // aka stream_executor::Platform::Id
 } SE_PlatformId;
 
 #define SE_PLATFORMID_STRUCT_SIZE TF_OFFSET_OF_END(SE_PlatformId, id)
 
-typedef struct SE_TimerFns {
- size_t struct_size;
- uint64_t (*nanoseconds)(SE_Timer timer);
- uint64_t (*microseconds)(SE_Timer timer);
-} SE_TimerFns;
-
-#define SE_TIMER_STRUCT_SIZE TF_OFFSET_OF_END(SE_Timer, microseconds)
```

Comment:
We do actually need SE_TimerFns. I forgot to add usage for it. But it should be created along with timer in `create_timer` and destroyed in `destroy_timer` along with SE_Timer. Initially I had these callbacks in SE_Timer, but then I split functions to SE_TimerFns (following your advice) but forgot to update usage.
Thank you for all the fixes!

Reply:
I've added it back and also used them in `create/destroy_timer`, please have a look!

Reply:
Thanks a bunch!